tools/Customization/DevHome.Customization/Views/OptimizeDevDriveDialog.xaml

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright (c) Microsoft Corporation.. -->
+<!-- Licensed under the MIT License. -->
+
+<ContentDialog
+    x:Class="DevHome.Customization.Views.OptimizeDevDriveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:environments="using:DevHome.Customization.ViewModels.Environments" d:DataContext="{d:DesignInstance Type=environments:DevDriveOptimizerCardViewModel}"
+    x:Uid="ms-resource:///DevHome.Customization/Views/OptimizeDevDriveDialog"
+    mc:Ignorable="d"
+    CornerRadius="8"

```

Comment:
The reason that the background smoke is the wrong color and the reason you have to set this to get round corners is because you need to set the style:
```suggestion
    Style="{StaticResource DefaultContentDialogStyle}"
```